package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lightninglabs/lnc-web",
-  "version": "0.3.4-alpha",
+  "version": "0.3.5-alpha",
```

Comment:
Ah sorry, missed that and what I mean that this specific change should be in a separate commit that bumps the version only. We want that commit to be the last commit before the merge commit, as we don't want any additional changes to have happened in the commit tree after the version has been bumped. This is currently placed as the second last commit. 

Hope that makes it more clear :)!

Reply:
That makes sense! Thanks for the clarification!

I've addressed this PR with that structure -- I hope it looks ok now 🙏